src/types/check_config.rs

```diff
@@ -26,7 +26,7 @@ pub const MAX_CHECK_INTERVAL_SECS: usize = CheckInterval::SixtyMinutes as usize;
 
 /// The CheckConfig represents a configuration for a single check.
 #[serde_as]
-#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
need this for tests?

Reply:
Yeah, makes it easier to serialize configs and do a full test